tests/codegen/riscv-abi/riscv64-lp64f-lp64d-abi.rs

```diff
@@ -4,7 +4,7 @@
 //@ only-linux
 #![crate_type = "lib"]
 
-// CHECK: define void @f_fpr_tracking(float %0, float %1, float %2, float %3, float %4, float %5, float %6, float %7, i8 zeroext %i)
+// CHECK: define void @f_fpr_tracking(float %0, float %1, float %2, float %3, float %4, float %5, float %6, float %7, i8 noundef zeroext %i) unnamed_addr #0
```

Comment:
The largely matches the loongarch64 code: https://github.com/rust-lang/rust/blob/550d1b4fb6de23990f4108815c3b1a9d1659e5c4/tests/codegen/loongarch-abi/loongarch64-lp64d-abi.rs#L13-L26